### PR TITLE
Update and unify formatting for a bunch of files

### PR DESCRIPTION
**Pull request by sethrj**
I've removed some of the `\brief` sitting around from older code pieces, as well as extra line separators (@pcanal ). I've also replaced `// >>> FOO` comments with `/// FOO ///` because they look less like kernel launches and conflict markers, and more like separators.

### CODE REVIEW

File: src/io/ImportPhysicsVector.hh [hunk 1]
```diff
@@ -4,7 +4,7 @@
 // SPDX-License-Identifier: (Apache-2.0 OR MIT)
 //---------------------------------------------------------------------------//
 //! \file ImportPhysicsVector.hh
-//! \brief Store Geant4 physics vector data
+//! Store Geant4 physics vector data.
```
Comment (stognini) [inline src/io/ImportPhysicsVector.hh:7]: I suggest removing the term "Geant4" for consistency. I changed the comments when refactoring the names and missed this one.

```suggestion
//! Store physics vector data.
```

File: src/io/ImportPhysicsVectorType.hh [hunk 2]
```diff
@@ -4,7 +4,7 @@
 // SPDX-License-Identifier: (Apache-2.0 OR MIT)
 //---------------------------------------------------------------------------//
 //! \file ImportPhysicsVectorType.hh
-//! \brief Geant4 PhysicsVector type enumerator
+//! Geant4 PhysicsVector type enumerator.
```
Comment (stognini) [inline src/io/ImportPhysicsVectorType.hh:7]:
```suggestion
//! PhysicsVector type enumerator.
```

**Comment by pcanal** [comment]
> I've removed some of the \brief sitting around from older code pieces

Why?  I.e. isn't that needed to generate doxygen version of the doc? (and/or are we using something else)

**Comment by sethrj** [comment]
@pcanal See http://llvm.org/docs/CodingStandards.html#doxygen-use-in-documentation-comments : there's an option in Doxygen (JAVADOC_AUTOBRIEF) that automatically marks the first sentence as the brief description. [The LLVM maintainers think](https://groups.google.com/g/llvm-dev/c/PrUnSeHfzMU?pli=1) (and I agree) that the brief adds visual clutter.

**Comment by pcanal** [comment]
@sethrj Fair enough :)

**Comment by stognini** [comment]
by looking at the list of files there are still some `\brief` still scattered around (and I bet there are some in non-modified files that are not listed here on github).
I also see that some `.hh` have their brief message after after `\file ...` but other do not. I assume that for very simple headers, such as a struct, the message is deemed unnecessary, but for more complex classes, this is still valuable. But please correct me if that is not the case.

**Comment by sethrj** [comment]
@stognini Thanks, I thought I'd grepped for the `\brief` but missed a couple. The`\file` documentation is generally redundant/pointless for "class" files -- the single class (and possibly related free functions) is the purpose of the file. For utility-type files with multiple related functions it's good to document exactly what they all have in common.

**Comment by sethrj** [comment]
OK, last briefs fixed and such.

**Review by stognini — APPROVED** [review]

**Pull request merged**